richreview/osfs_utils.py

```diff
@@ -1,55 +0,0 @@
-""" this module loads webapp on the static file system, and serves to the XBlock """
-
-
-import re
-import os
-
-def osfs_mkdir(fs, dst_path):
-    sup = os.path.dirname(dst_path)
-    if not fs.exists(sup):
-        osfs_mkdir(fs, sup)
-        fs.makedir(sup)
```

Comment:
`fs.makedir` takes a recursive argument that I think would handle this, wouldn't it?